internal/runtime/client/client.go

```diff
@@ -488,8 +488,13 @@ func httpCall(ctx context.Context, request, response runtime.Object, opts *httpC
 	})
 
 	resp, err := client.Do(httpRequest)
+	respStatus := "Unknown"
```

Comment:
Thinking about setting status to `""` (empty) when we don't have a status 🤔 Or at least also add the type to the API definition and use it from there?

Reply:
I personally don't like an idea of an empty label, because it doesn't add any meaning or value to grafana dashboards or alerts (e.g. labels are often used in grafana panel legend. Empty value in the legend will be very confusing)

Reply:
I thought about adding it to API definition, but I really don't know if it is correct - because this value won't be used in code, it will be only used in metrics

Reply:
(as proposed below) should we move the typecast and handling into the observe function and also add this as a const at the metrics library instead?

Reply:
@chrischdi done :)